calculation-service/src/main/java/org/java/mentorship/calculation/CalculationApplication.java

```diff
@@ -0,0 +1,15 @@
+package org.java.mentorship.calculation;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.PropertySource;
+
+@SpringBootApplication
+@PropertySource("classpath:task.properties")
```

Comment:
Any chance we could put this annotation on the SpringConfig class?
Generally we leave application classes as lightweight as possible